fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -567,6 +567,20 @@ ACTOR Future<Void> releaseResolvingAfter(ProxyCommitData* self, Future<Void> rel
 	return Void();
 }
 
+// Try to identify recovery transaction and backup's apply mutations (blind writes).
+// Both cannot be rejected and are approximated by looking at first mutation
+// starting with 0xff.
+bool canReject(const std::vector<CommitTransactionRequest>& trs) {
+	for (const auto& tr : trs) {
+		if (tr.transaction.mutations.empty()) continue;
+		if (tr.transaction.mutations[0].param1.startsWith(LiteralStringRef("\xff")) ||
```

Comment:
The idea here is that the recovery transaction will definitely have a first mutation starting with `\xff`, so we will never reject it, and it's ok if we let other system transactions pass through because they are a small fraction and letting them go only costs us resources? I'm not that familiar with the contents of the recovery transaction, is it challenging to be more accurate?

Reply:
Yes. I try to make the cost of this function small and it's ok to let other system transaction go through.

Recovery transaction can start with different keys, so I didn't try very hard to do exact match.